Formula/orientdb.rb

```diff
@@ -39,4 +39,10 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    # quick check if file is there
```

Comment:
Can do this with `assert File.exist?` instead. It'dbe preferable to do something with e.g. `orientdb-console` instead?